config/default_records/tag_groups/005_ultima.yml

```diff
@@ -0,0 +1,199 @@
+Ultima P1:
+  visible: true
```

Comment:
No adapter_type given

Reply:
From what I can tell, adapter_type is used to filter tag groups. So if you want all the relevant ones for the Chromium type of tagging chemistry you can filter to the ones with 'Chromium' adapter type.
Here maybe you could just set it to 'Ultima' for now.